tests/gold_tests/bigobj/bigobj.test.py

```diff
@@ -52,65 +47,88 @@
 )
 
 ts.Disk.remap_config.AddLine(
-    'map https://localhost http://localhost'
+    f'map https://localhost:{ts.Variables.ssl_port} http://localhost:{ts.Variables.port}'
+)
+ts.Disk.remap_config.AddLine(
+    f'map https://localhost:{ts.Variables.ssl_portv6} http://localhost:{ts.Variables.port}'
 )
-
-# Set up to check the output after the tests have run.
-#
-log_id = Test.Disk.File("log2.txt")
-log_id.Content = "log2.gold"
 
 # Size of object to get.  (NOTE:  If you increase this significantly you may also have to increase cache
 # capacity in tests/gold_tests/autest-size/min_cfg/storage.config.  Also, for very large objects, if
 # proxy.config.diags.debug.enabled is 1, the PUSH request will timeout and fail.)
 #
 obj_kilobytes = 10 * 1024
+obj_bytes = obj_kilobytes * 10
+header = "HTTP/1.1 200 OK\r\nContent-length: {}\r\n\r\n".format(obj_bytes)
+
+
+def create_pushfile():
+    f = open(Test.RunDirectory + "/objfile", "w")
+    f.write(header)
+    f.write("x" * obj_bytes)
+    f.close()
+    return True
+
 
 tr = Test.AddTestRun("PUSH an object to the cache")
 # Delay on readiness of TS IPv4 ssl port
-tr.Processes.Default.StartBefore(ts)
-#
+tr.Processes.Default.StartBefore(ts, ready=lambda: create_pushfile())
```

Comment:
Does this augment or replace the default ready test?

Reply:
This augments the default port ready test as far as I can tell @dragon512 can you verify that?  It looks like there is an obj ready test which I assume includes the port ready for ts as well as the explicitly specified ready function.